client/webpack.config.js

```diff
@@ -56,6 +56,31 @@ const defaultBrowserslistEnv = isCalypsoClient && ! isDesktop ? 'evergreen' : 'd
 const browserslistEnv = process.env.BROWSERSLIST_ENV || defaultBrowserslistEnv;
 const extraPath = browserslistEnv === 'defaults' ? 'fallback' : browserslistEnv;
 
+function filterEntrypoints( entrypoints ) {
+	if ( ! process.env.ENTRY_LIMIT ) {
+		return entrypoints;
+	}
+
+	const allowedEntrypoints = process.env.ENTRY_LIMIT.split( ',' );
+	// eslint-disable-next-line no-console
+	console.warn( '[entrylimit] Only building entrypoints %s', allowedEntrypoints.join( ', ' ) );
+	const allowed = {};
+	let buildingAtLeastOne = false;
+	Object.entries( entrypoints ).forEach( ( [ key, val ] ) => {
+		if ( allowedEntrypoints.includes( key ) ) {
+			allowed[ key ] = val;
+			buildingAtLeastOne = true;
+		}
```

Comment:
Should we also `console.warn` when someone has input an incorrect entry point name? I'm assuming that a common mistake would be to input `ENTRY_LIMIT=jetpack-cloud` instead of `ENTRY_LIMIT=entry-jetpack-cloud`

Reply:
done!